pyiron_contrib/atomistic/randspg/randspg.py

```diff
@@ -53,6 +55,26 @@ def list_structures(self):
         else:
             return []
 
+    def _executable_activate(self, enforce=False):
```

Comment:
Is overriding this still necessary after the move?

Reply:
You are right - this was part of the debugging.